rules/import.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  rules: {
+    // Ref: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md
+    // Enforces the addition of file extensions when importing.
+    'import/extensions': [
+      'error',
+      'never',
+      {
+        mp3: 'always',
+        mp4: 'always',
+        svg: 'always',
+        png: 'always',
+      },
```

Comment:
how about (.jpg, .jpeg, css, scss, sass, less)

Reply:
Actually I did not think it thru. Just made it on use cases what we have.
fiy, this is airbnb
```
    'import/extensions': ['error', 'ignorePackages', {
      js: 'never',
      mjs: 'never',
      jsx: 'never',
    }],
```
Might we should go with this. @beermindnor @tavurth 
```
    'import/extensions': ['error', 'always', {
      js: 'never',
      mjs: 'never',
      jsx: 'never',
    }],
```

Reply:
the second, does it mean we need to put extension for dependency?
import BarLoader from 'loaders/barloader.js'; ? if yes then we should go with airbnb?

Reply:
Second option says that we always define extension on import except 'js', 'jsx', 'mjs'.